docs/build/web.md

```diff
@@ -127,25 +127,25 @@ Q: I have a C++ project for web scenario, which runs a ML model using ONNX Runti
 
 > One important note is that ONNX Runtime has dependencies on lots of 3rd party libraries such as protobuf, onnx, and others. You may need copy necessary header files to your project. You also take care of cases of library version conflicts or emsdk version conflicts between ONNX Runtime and your project.
 
-## Build onnxruntime-web (NPM package)
+# Build onnxruntime-web (NPM package)
```

Comment:
should not promote the hierarchy

the current hierarchy structure is:

---------

(#) Build ONNX Runtime for Web
   - ( ## ) Build ONNX Runtime for WebAssembly
   - ( ## ) Build onnxruntime-web (NPM package)

-------------

with this change, it will become:


--------
(#) Build ONNX Runtime for Web
 - ( ## ) Build ONNX Runtime for WebAssembly

(#) Build onnxruntime-web (NPM package)

----------------

The idea is, building ORT for web is composed by 2 steps : build webassemby and build npm package. so we should keep the current hierarchy structure

Reply:
The title may be a little confusing, so we can change to a better title if you have any good idea

Reply:
Thanks for catching this - I changed the hierarchy to support the link to downloading the WASM artifacts.
Updated the doc